tests/conftest.py

```diff
@@ -123,7 +124,7 @@ def create_client(self, term_command='sleep infinity', title=None):
         command = ['xterm'] + title + ['-class', wmclass, '-e', 'bash', '-c', term_command]
 
         # enforce a hook when the window appears
-        self.call(['rule', 'once', 'class='+wmclass, 'hook=here_is_'+wmclass])
+        self.call(['rule', 'once', 'class=' + wmclass, 'hook=here_is_' + wmclass])
```

Comment:
Since we're already using fstrings anyways, this could be:

```suggestion
        self.call(['rule', 'once', f'class={wmclass}', f'hook=here_is_{wmclass}'])
```

instead.

Reply:
I had the impression that one should avoid fstrings. Here, I find the non-fstring version more readable (also in the other changeset), so I'd keep it as it is.